roles/thelounge/tasks/main.yaml

```diff
@@ -0,0 +1,48 @@
+- name: Install TheLounge package
+  ansible.builtin.package:
+    name: thelounge
+    state: present
+
+- name: Install acl package
+  ansible.builtin.package:
+    name: acl
+    state: present
+
+- name: Set up SSL certificate ACL
+  ansible.builtin.acl:
+    path: '{{ item }}'
+    entity: thelounge
+    etype: user
+    permissions: r
+    state: present
+  loop:
+    - '{{ thelounge__https_key }}'
+    - '{{ thelounge__https_certificate }}'
+  when:
+    - thelounge__https_key != ''
+    - thelounge__https_certificate != ''
```

Comment:
How I can do it also for parent directories?

Reply:
You would need to recursively test if the user can access each directory in the chain...

I think this might be getting out of scope for the role, and maybe it would be better to deal with all this in the playbook or even in the client configuration. Already installing the acl package seems a bit much here, IMHO

Reply:
Leaving for a follow-upv